skimage/restoration/deconvolution.py

```diff
@@ -105,7 +105,7 @@ def wiener(image, psf, balance, reg=None, is_real=True, clip=True):
 
            https://www.osapublishing.org/josaa/abstract.cfm?URI=josaa-27-7-1593
 
-           http://research.orieux.fr/files/papers/OGR-JOSA10.pdf
+           https://hal.archives-ouvertes.fr/hal-00674508/document
```

Comment:
```suggestion
           https://hal.archives-ouvertes.fr/hal-00674508
```
(Taking the reader only to the landing page, where they can easily download the PDF file...)